Docs/MoltenVK_Runtime_UserGuide.md

```diff
@@ -288,6 +288,7 @@ In addition to core *Vulkan* functionality, **MoltenVK**  also supports the foll
 - `VK_EXT_debug_utils`
 - `VK_EXT_fragment_shader_interlock` *(requires Metal 2.0 and Raster Order Groups)*
 - `VK_EXT_host_query_reset`
+- `VK_EXT_image_robustness`
```

Comment:
Hmmm...someday I'll have to mull over possible mechanisms for generating the lists in `MoltenVK_Runtime_UserGuide.md` and `Whats_New.md` from `MVKExtensions.def`.